webui/react/src/services/api.ts

```diff
@@ -163,10 +163,18 @@ export const killTensorboard = generateDetApi<
   CommandIdParams, Api.V1KillTensorboardResponse, void
 >(Config.killTensorboard);
 
+export const getTemplates = generateDetApi<
```

Comment:
**nit:** Not 100% sure on this but I think there are a couple of different types of template (not in the WebUI yet) but might be better to be explicit now and just call it `getTaskTemplates`?